includes/admin/helper/class-wc-helper.php

```diff
@@ -466,9 +466,10 @@ private static function _filter( &$subscriptions, $filter ) {
 	public static function admin_enqueue_scripts() {
 		$screen       = get_current_screen();
 		$screen_id    = $screen ? $screen->id : '';
-		$wc_screen_id = sanitize_title( __( 'WooCommerce', 'woocommerce' ) );
 
-		if ( $wc_screen_id . '_page_wc-addons' === $screen_id && isset( $_GET['section'] ) && 'helper' === $_GET['section'] ) {
+		if ( 'marketplace_page_wccom-subscriptions' == $screen_id || (
```

Comment:
Can we improve the readability of this statement a bit? Perhaps breaking it up into more one liners? You were heading in the right direction.

Reply:
Reverted this now as we're back to `woocommerce` screen id.